### DISCUSSION TITLE
220MHz full power

### DISCUSSION THREAD

**Discussion by KQ4GAH** [egzumer/uv-k5-firmware-custom#473] [Q&A]
Is there a way to increase the power output of the radio on the 220 band?

**Comment by OE1MWW** [comment]
as the design is built for 2m & 70cm, you have to redesign and
modify the RX and TX filters. Firmware can not help on this.

**Comment by ErikS-web** [comment]
YES **_real HAM hobby._** redesign, soldering etc ....

_No firmware-update necessary_ 👍

**Comment by egzumer** [comment]
Power settings are configured in power calibration table in eeprom + power calculation routines in the firmware. You can play with that but you probably shouldn't as mentioned above.
